docs/getting-started/installation.md

```diff
@@ -17,37 +18,162 @@ This can be done with the following command:
     
 `/usr/sbin/softwareupdate --install-rosetta --agree-to-license`
 
-## Virtual Environment (Optional)
+
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.box(
+        rx.alert_title("Error ", rx.code("command not found: reflex"),),
+        rx.alert_description(
+            "If you install Reflex with no virtual environment and get this error it means your PATH cannot find the reflex package. A virtual environment should solve this problem.",
```

Comment:
This problem can also be solved by running `python -m reflex` instead of `reflex`... not sure if that's worth mentioning

Reply:
good point